### PR TITLE
[yamahareceiver] Remove Apache commons libraries

### PR DESCRIPTION
**Pull request by mlobstein**
Remove usage of org.apache.commons.lang3.ArrayUtils and org.apache.commons.lang3.StringUtils per #7722

### CODE REVIEW

File: bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithPresetControlXML.java [hunk 1]
```diff
@@ -156,7 +154,7 @@ private int convertToPresetNumber(String presetValue) {
                     if (Character.isLetter(presetAlpha) && Character.isUpperCase(presetAlpha)
                             && Character.isDigit(presetValue.charAt(1))) {
                         int presetNumber = Integer.parseInt(presetValue.substring(1));
-                        return (ArrayUtils.indexOf(LETTERS, presetAlpha) + 1) * 100 + presetNumber;
+                        return (new String(LETTERS).indexOf(presetAlpha) + 1) * 100 + presetNumber;
```
Comment (wborn) [inline bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithPresetControlXML.java:157]: You could probably prevent some garbage collection by turning `LETTERS` into a String itself.
Then you can use `char presetAlpha = LETTERS.charAt(..)` below.

**Review by wborn — COMMENTED** [review]
It looks good to me! I see one small improvement:

File: bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithPresetControlXML.java (hunk 1 above)
Comment (mlobstein) [reply bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithPresetControlXML.java]: Good idea. I have made the change. Thanks

**Review by wborn — APPROVED** [review]
Thank you!

**Pull request merged**